ymcatwincities/openy
# Add @font-your-face module

## Steps for review

- Open Appearance -> @font-your-face -> Custom Fonts.
- Add Custom Font with the file in WOFF format.
- Check if the font appears on site.
